langfuse/openai.py

```diff
@@ -505,7 +530,7 @@ async def _wrap_async(
     start_time = _get_timestamp()
     arg_extractor = OpenAiArgsExtractor(*args, **kwargs)
 
-    generation = _get_langfuse_data_from_kwargs(
+    generation, is_nested_trace = _get_langfuse_data_from_kwargs(
```

Comment:
Was wondering if we could return a nested function bool from the existing function which would make the implementation easier to review and also possibly reduce the touch points in the code. If we change, we change on ly inside the function and rest should fall in place.